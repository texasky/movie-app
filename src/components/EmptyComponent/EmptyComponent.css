.empty-component {
    margin-top: 130px;
}