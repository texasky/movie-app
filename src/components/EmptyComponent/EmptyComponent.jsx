import './EmptyComponent.css'

const EmptyComponent = (props) => {
    return (
        <div className="empty-component" data-testid="empty-component">
            <span>{props.text}</span>
        </div>
    )
}

export default EmptyComponent;