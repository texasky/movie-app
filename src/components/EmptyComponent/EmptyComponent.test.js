import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import EmptyComponent from './EmptyComponent';

test('Empty component tests', () => {
    const utils = render(<EmptyComponent text="Some text to test"/>);

    const emptyComponent = utils.getByTestId('empty-component');

    expect(emptyComponent).toBeInTheDocument();
    expect(emptyComponent).toHaveTextContent('Some text to test');
});

test('Back button component matches snapshot', () => {
    const tree = renderer.create(<EmptyComponent text="Some text to test"/>).toJSON();
    expect(tree).toMatchSnapshot();
});