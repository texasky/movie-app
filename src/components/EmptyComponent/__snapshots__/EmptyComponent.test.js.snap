// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Back button component matches snapshot 1`] = `
<div
  className="empty-component"
  data-testid="empty-component"
>
  <span>
    Some text to test
  </span>
</div>
`;
